app/Http/Controllers/Auth2/RegisterController.php

```diff
@@ -0,0 +1,73 @@
+<?php
```

Comment:
Whats the difference of `Auth2` and `Auth` folders?

Reply:
The Auth2 folder was actually a php artisan authentication scaffolding or auto generated Authentication which was originally named Auth folder. I was confused at that time if I were to use the generated authentication or make my I own authentication controllers. Which I eventually made my own controllers for the authentication and I recently delete the Auth2 folder. Sorry ma'am, for including the Auth2 folder in the commit .